puppet/zulip/manifests/base.pp

```diff
@@ -48,6 +48,7 @@
              'missedmessage_mobile_notifications',
              'outgoing_webhooks',
              'signups',
+             'slash_commands',
```

Comment:
Do you also need to update `puppet/zulip_ops/files/nagios3/conf.d/services.cfg` the way you did for the outgoing webhooks queue?

(See https://github.com/zulip/zulip/commit/8881b5eb9f499312a5ea192475669acdcfbf7a41)